packages/components/src/styles/index.css

```diff
@@ -9,5 +9,6 @@
 @import "../components/primitives/Tooltip/Tooltip.css";
 @import "../components/primitives/Modal/Modal.css";
 @import "../components/primitives/form/RadioButton/RadioButton.css";
+@import "react-toastify/dist/ReactToastify.css";
```

Comment:
is this needed?

Reply:
We could create our own css but I prefer to overwrite a few properties of the base css provided by react-toastify